src/super_gradients/training/utils/sg_trainer_utils.py

```diff
@@ -459,27 +467,37 @@ def get_lr_info(model: nn.Module, param_groups: List[Dict[str, Union[str, float,
     total_params = sum(p.numel() for p in model.parameters())
     optimized_params = sum(p.numel() for group in param_groups for p in group["params"])
 
-    info_str = f"    - Model: {type(model).__name__}  ({total_params / 1e6:.1f}M parameters"
+    info_str = f"    - Model: {type(model).__name__}  ({_format_value(total_params)} parameters"
+    info_str += f", {_format_value(optimized_params)} optimized)\n"
+    info_str += "    - Learning Rates and Weight Decays:\n"
 
-    if optimized_params >= 1e6:
-        precision_optimized = max(0, 4 - int(optimized_params / 1e6).bit_length())
-        info_str += f", {optimized_params / 1e6:.{precision_optimized}f}M optimized)\n"
-    else:
-        precision_optimized = max(0, 4 - int(optimized_params).bit_length())
-        info_str += f", {optimized_params:.{precision_optimized}f}M optimized)\n"
+    lr_wd_groups = {}
 
-    info_str += "    - Learning rates:\n"
     for group in param_groups:
         group_name = group["name"]
         group_lr = group["lr"]
+        group_wd = group["weight_decay"]
         group_params = sum(p.numel() for p in group["params"])
 
-        if group_params >= 1e6:
-            precision_group = max(0, 4 - int(group_params / 1e6).bit_length())
-            info_str += f"      - {group_name}: {group_lr} ({group_params / 1e6:.{precision_group}f}M parameters)\n"
+        if group_name not in lr_wd_groups:
+            lr_wd_groups[group_name] = {"params": 0, "lr_params": {}, "wd_params": {}}
+        lr_wd_groups[group_name]["params"] += group_params
+
+        if group_lr not in lr_wd_groups[group_name]["lr_params"].keys():
+            lr_wd_groups[group_name]["lr_params"][group_lr] = group_params
+        else:
+            lr_wd_groups[group_name]["lr_params"][group_lr] += group_params
+
+        if group_wd not in lr_wd_groups[group_name]["wd_params"].keys():
+            lr_wd_groups[group_name]["wd_params"][group_wd] = group_params
         else:
-            precision_group = max(0, 4 - int(group_params).bit_length())
-            info_str += f"      - {group_name}: {group_lr} ({group_params:.{precision_group}f}M parameters)\n"
+            lr_wd_groups[group_name]["wd_params"][group_wd] += group_params
+
+    for group_name, info in lr_wd_groups.items():
+        all_group_params = info["params"]
+        lr_str = ", ".join([f"LR: {lr_val} ({_format_value(lr_params)} parameters)" for lr_val, lr_params in info["lr_params"].items()]) + ". "
+        wd_str = ", ".join([f"WD: {wd_val}, ({_format_value(wd_params)} parameters)" for wd_val, wd_params in info["wd_params"].items()]) + ". "
+        info_str += f"      - {group_name}: ({_format_value(all_group_params)} parameters). {lr_str} {wd_str}\n"
```

Comment:
Maybe add left padding for group name to maximum length of parameter groups (to make them all aligned vertically during print)?